libr/core/graph.c

```diff
@@ -720,27 +633,23 @@ static int graph_refresh(struct graph_refresh_data *grd) {
 	return R_TRUE;
 }
 
-static void graph_init(struct graph *g) {
-	g->nodes = NULL;
-	g->edges = NULL;
-
+static void agraph_init(AGraph *g) {
 	g->is_callgraph = R_FALSE;
 	g->is_instep = R_FALSE;
 	g->is_simple_mode = R_TRUE;
 	g->is_small_nodes = R_FALSE;
 	g->need_reload_nodes = R_TRUE;
-	g->curnode = 0;
-	g->need_update_seek = R_TRUE;
-	g->update_seek_on = g->curnode;
+	g->curnode = NULL;
+	g->update_seek_on = NULL;
 	g->force_update_seek = R_TRUE;
-
-	ostack_init(&g->ostack);
+	g->history = r_stack_new (16);
```

Comment:
No reason actually. It's just a starting value. RStack accept an initial capacity in the case you already know how big your stack can be. It's probably better to use a constant INITIAL_STACK_CAPACITY.